openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -526,6 +533,10 @@ This specification defines the following request parameters that can be supplied
 
 Note: When processing the Authorization Request, the Credential Issuer MUST take into account that the `issuer_state` is not guaranteed to originate from this Credential Issuer in all circumstances. It could have been injected by an attacker.
 
+Additional Authorization Request parameters MAY be defined and used,
```

Comment:
do we really need to state this again?

Reply:
Definitely not technically necessary, but I think doing so will help implementers that aren't that familiar with OAuth2.